lib/utils/jsontools.go

```diff
@@ -70,10 +70,19 @@ func FastUnmarshal(data []byte, v interface{}) error {
 	return nil
 }
 
+// SafeConfig uses jsoniter's ConfigFastest settings but enables map key
+// sorting to ensure CompareAndSwap checks consistently succeed.
+var SafeConfig = jsoniter.Config{
+	EscapeHTML:                    false,
+	MarshalFloatWith6Digits:       true, // will lose precession
```

Comment:
[Precession is a change in the orientation of the rotational axis of a rotating body.](https://en.wikipedia.org/wiki/Precession)

I realize that this is taken right from the definition of `jsoniter.ConfigFastest` but maybe we don't need to also mirror the misspelled comments 😅

Reply:
oh geez I'm embarrassed that I missed the typo, will fix!